generation/hermetic_library_generation.sh

```diff
@@ -102,7 +109,13 @@ docker run \
   --repository-path="${workspace_name}"
 # commit the change to the pull request.
 git add java-* pom.xml gapic-libraries-bom/pom.xml versions.txt "${generation_config}"
-git commit --allow-empty -m "${message}"
+changed_files=$(git diff --cached --name-only)
+if [[ "${changed_files}" == "" ]]; then
+    echo "Nothing to commit, exit."
```

Comment:
Can we provide a more verbose message? Something like "There is no generated code change with the generation config change {insert config diff}."
In addition, this should be a very rare case, right? The only case I can think of is there are googleapis committish change but the change is not for Java.

Reply:
> Something like "There is no generated code change with the generation config change {insert config diff}."

Done.

> In addition, this should be a very rare case, right? The only case I can think of is there are googleapis committish change but the change is not for Java.

Yes, it is a rare case.